web-console/conceptual-designs/topology.md

```diff
@@ -0,0 +1,15 @@
+# Topology View
+- A topology overview page will enable administrators and developers to easily visualize their applications, to mentally organize application components, and to get a better insight into the health and connections of those components.
+
+## Use Cases
+1. As a system administrator, I want to:
+  - See how the components of my project(or projects) are connected together.
+  - Verify that the components of my project are all present and functioning correctly.
+  - See which parts of my project are connected to external components.
+1. As a developer, I want to:
+  - Get quick access to information about my deployments and their component parts.
+  - Easily view pod status for a given deployment and view logs.
+  - Narrow my focus to a specific set of components based on label or namespace criteria.
+
+## Design
+- [The Invision prototype linked here includes designs and flows for the use cases described above.](https://redhat.invisionapp.com/share/2ULBOIBJT4M#)
```

Comment:
@cshinn should there be hotspots on your invision flow? Of course, somewhat unrelated to this PR!